apps/barista-design-system/src/components/ba-decision-graph/ba-decision-graph-node/ba-decision-graph-node.scss

```diff
@@ -0,0 +1,144 @@
+@import 'interactive-common';
+@import 'colors';
+@import 'font-mixins';
+
+:host {
+  max-width: 100%;
+}
+
+::ng-deep button {
+  @include dt-interactive-reset();
+  @include dt-main-font();
+}
+
+::ng-deep .dt-uxd-node-content p {
+  margin-bottom: 0;
+}
+
+.dt-uxd-node {
+  height: auto;
+  // width: 100%;
+  border-top: 2px solid #00a1b2;
+  border-bottom: 2px solid #00a1b2;
+  margin: 4px;
+}
+
+.dt-uxd-flex {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  flex-wrap: nowrap;
+  flex-shrink: 0;
+}
+
+.dt-uxd-node-grid {
+  display: grid;
+  grid-template-columns: 1fr auto;
+}
+
+.dt-uxd-node-content {
+  width: fit-content;
+  margin: 4px;
+  padding: 4px;
+  border-radius: 2px;
+}
+
+.dt-uxd-node-edges {
+  max-height: 36px;
+  width: fit-content;
+  margin: 4px auto;
+}
+
+.dt-uxd-button {
+  height: 32px;
+  min-width: 40px;
+  border-radius: 3px;
+  margin: 0 4px;
+  color: white;
+
+  &:disabled {
+    background-color: $gray-300;
+
+    &:hover {
+      background-color: $gray-300;
+    }
+  }
+}
+
+.dt-uxd-yes {
+  background-color: $purple-600;
+
+  &:hover {
+    background-color: $purple-700;
+  }
+
+  &:active {
+    background-color: $purple-800;
+  }
+}
+
+.dt-uxd-no {
+  background-color: $red-500;
+
+  &:hover {
+    background-color: $red-600;
+  }
+
+  &:active {
+    background-color: $red-700;
+  }
+}
+
+.dt-uxd-got-it {
+  background-color: $purple-600;
+
+  &:hover {
+    background-color: $purple-700;
+  }
+
+  &:active {
+    background-color: $purple-800;
+  }
+}
+
+.dt-uxd-start-over-button {
+  background-color: $turquoise-500;
+
+  &:hover {
+    background-color: $turquoise-600;
+  }
+
+  &:active {
+    background-color: $turquoise-700;
+  }
+}
+
+.dt-uxd-back-button {
+  background-color: $purple-600;
+
+  &:hover {
+    background-color: $turquoise-600;
+  }
+
+  &:active {
+    background-color: $turquoise-700;
+  }
+}
```

Comment:
I know that styling is not part of this PR, but try to not repeat yourself. 
if hover and active are the same as on a default button, try and group them in a `dt-uxd-navigation-button` and apply the overrides for the color on the back button for example.